test/test_main.py

```diff
@@ -3,6 +3,8 @@
 import pyega3.pyega3 as pyega3
 
 
+# FIXME bjuhasz, afoix
```

Comment:
Here's a test which might be a blueprint for fixing this test:

```
    def test_main_without_proper_arguments_exits_cleanly(self):
        with mock.patch('sys.argv', return_value=None):
            with mock.patch('builtins.input', return_value='test_username'):
                with mock.patch('getpass.getpass', return_value='test_password'):
                    self.client.main()
```

But maybe this^^^ test above tests something else than the original `test_main_exits` test.